### ISSUE TITLE
Where are the Julia communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/julia#110]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Julia supported by one or more large organizations?
* Does Julia have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Julia? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Julia? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Julia? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Julia?
* Are there popular newsletters for Julia?
* Is Julia taught at programming bootcamps? (If so, what are some examples?)
* Is Julia taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Julia?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by SaschaMann** [comment]
> Is Julia supported by one or more large organizations?

[NumFocus](https://www.numfocus.org/sponsored-projects) and [Julia Computing](https://juliacomputing.com/).

> Does Julia have an official community manager?

Alex Arslan (`@ararslan`). Jan Herriman (`@xorJane`) is responsible for Diversity and Outreach at Julia Computing, she hosts regular intro livestreams for example.

> Do you know of specific communities (online or offline) that are enthusiastic about Julia? (Chat communities, forums, meetups, student clubs, etc)

The official Julia [Discourse](https://discourse.julialang.org) and [Slack](http://julialang.slack.com/) are the biggest communities that I know of. There are various User Groups and Meetups listed on the [website](https://julialang.org/community/) but I am not sure how active they are.

> Are there popular conferences for Julia? (If so, what are some examples?)

[JuliaCon](http://juliacon.org/) is the major one. Other than that, conferences with people from the scientific community, especially computational sciences, often have Julia users attending. I know that there was a small meetup at March Meeting and I think there was one at the DPG spring conferences as well.

> Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Julia? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)

I don't know.

> Are there specific groups or programs dedicated to mentoring people in Julia?

Google Summer of Code.

> Are there popular newsletters for Julia? / Is Julia taught at programming bootcamps? (If so, what are some examples?)

I don't know any.

> Is Julia taught at universities? (If so, what are some examples?)

It's used for teaching computational sciences and numerical mathmatics at quite a few universities. [https://julialang.org/teaching/](Here) is an incomplete list.

> In other words: where do people care a lot and/or know a lot about Julia?

A large part of the community is made up of scientists or students that use Julia in their research. People in Finance and economics are other groups that already use Julia or are waiting for a stable 1.0 release.

**Comment by kytrinyx** [comment]
This is superb, thanks so much!
